deb/common/control

```diff
@@ -82,7 +82,7 @@ Description: Docker CLI: the open-source application container engine
 
 Package: docker-ce-rootless-extras
 Architecture: linux-any
-Depends: docker-ce, ${shlibs:Depends}
+Depends: ${shlibs:Depends}
```

Comment:
Your PR title implies you might have intended to add `Enhances: docker-ce`? (this LGTM either way)

Reply:
Arf; edited the wrong copy of this file 😂 

Yes, I wanted to add 

```suggestion
Depends: ${shlibs:Depends}
Enhances: docker-ce
```

Mostly to indicate "hey this is to be used together with `docker-ce`"; that still LGTY if I change?